### ISSUE TITLE
how to explain the mask loss?

### ISSUE THREAD

**Issue by YujieOuO** [alibaba/animate-anything#52]
hi , i notice the finetune loss has been changed. 
for the mask loss:  loss += F.mse_loss(predict_x0*(1-mask), condition_latent*(1-mask))
could you please tell me how can this loss control the motion region?

**Comment by YujieOuO** [comment]
1. It seems use image latent to control the generation results of noisy latent, instead of using the processed GT.
What are the benefits of this loss?

2. The mask concatenated in input_latent has all zeros for its first frame. Wouldn't this cause a gap between training and infer?
